pages/docs/architecture.md

```diff
@@ -60,58 +60,58 @@ Below is a representative example from the Raspberry Pi family, which helps expl
 
 ### OpenVPN
 
-ResinOS will provide the user with an OpenVPN server that they might use. It is worth noting that this server will be disabled by default and manual interaction from the user is needed to activate and configure this server to their needs.
+BalenaOS will provide the user with an OpenVPN server that they might use. It is worth noting that this server will be disabled by default and manual interaction from the user is needed to activate and configure this server to their needs.
 
 ## Image Partition Layout
 
 <img src="/images/docs/arch/partition-layout.png" width="90%">
 
-The first partition, resin-boot, is meant to hold boot important bits according to each board (e.g. kernel image, bootloader image). It also holds a very important file that you will find mentioned elsewhere in this document (i.e. `config.json`). The config.json file is the central point of configuring ResinOS and defining its behaviour, for example you can set your hostname, allow persistent logging, etc.
-Resin-root is the partition that holds our read-only root filesystem; it holds almost everything that ResinOS is.
-Resin-update is an empty partition that is only used when the rootfs is to be updated. We follow the Blue-Green deployment strategy. Essentially we have one active partition that is the OS’s current rootfs and one dormant one that is empty, we download the new rootfs to the dormant partition and try to switch them, if the switch is successful the dormant partition becomes the new rootfs, if not, we rollback to the old active partition.
-Resin-conf is the partition that holds persistent data as explained in the [Stateless and Read-only rootfs](#stateless-and-read-only-rootfs).
-Resin-data is the partition that holds downloaded Docker images. Generally any container data will be found here.
+The first partition, balena-boot, is meant to hold boot important bits according to each board (e.g. kernel image, bootloader image). It also holds a very important file that you will find mentioned elsewhere in this document (i.e. `config.json`). The config.json file is the central point of configuring balenaOS and defining its behaviour, for example you can set your hostname, allow persistent logging, etc.
+Balena-root is the partition that holds our read-only root filesystem; it holds almost everything that balenaOS is.
+Balena-update is an empty partition that is only used when the rootfs is to be updated. We follow the Blue-Green deployment strategy. Essentially we have one active partition that is the OS’s current rootfs and one dormant one that is empty, we download the new rootfs to the dormant partition and try to switch them, if the switch is successful the dormant partition becomes the new rootfs, if not, we rollback to the old active partition.
+Balena-conf is the partition that holds persistent data as explained in the [Stateless and Read-only rootfs](#stateless-and-read-only-rootfs).
```

Comment:
We should actually double check, but I believe the partitions names are now `resin-boot`, `resin-rootA`, `resin-rootB`, `resin-state` and `resin-data`. Hopefully @agherzan can confirm

Reply:
That is correct, We haven't changed the partition names.